spec/controllers/users_controller_spec.rb

```diff
@@ -152,7 +152,8 @@
       before do
         create(:courses_user, user_id: user.id,
                               course_id: course.id,
-                              role: CoursesUsers::Roles::STUDENT_ROLE)
+                              role: CoursesUsers::Roles::STUDENT_ROLE,
+                              real_name: user.real_name)
```

Comment:
Is this necessary?

Reply:
In my local repo I don't have that. Maybe I need to force push.

Reply:
This change should be gone.